src/decoder/decoder-wrappers.cc

```diff
@@ -195,6 +195,87 @@ DecodeUtteranceLatticeFasterClass::~DecodeUtteranceLatticeFasterClass() {
   delete decodable_;
 }
 
+template <typename FST>
+bool DecodeUtteranceLatticeIncremental(
+    LatticeIncrementalDecoderTpl<FST> &decoder, // not const but is really an input.
```

Comment:
In general passing non-const reference is not allowed in Google style -- if it gets changed, then pass a pointer and move it to the end of the args list.